src/main/java/org/zalando/nakadi/controller/EventPublishingController.java

```diff
@@ -100,19 +106,23 @@ private ResponseEntity postEventInternal(final String eventTypeName,
         }
     }
 
+    private void reportSLOs(final long startingNanos, final int totalSizeBytes, final int eventCount) {
+        final long msSpent = TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startingNanos);
```

Comment:
It is not clear why we need ns in one other method, maybe we can refactor it a bit. What do you think?